drift/core/doublekl.py

```diff
@@ -87,7 +97,12 @@ def _ev_save_hook(self, f, evextra):
         kltransform.KLTransform._ev_save_hook(self, f, evextra)
 
         # Save out S/F ratios
-        f.create_dataset("f_evals", data=evextra["f_evals"])
+        if self.diagonalisation_order == "sf":
+            f_evals = evextra["f_evals"]
+        else:
+            f_evals = evextra["f_evals"][::-1]
```

Comment:
Why do you reverse the order for F/S eigenvalues? To be consistent with the single-KL F/S case, shouldn't the order be left as it is?